modules/services/sharing/src/main/kotlin/au/com/shiftyjelly/pocketcasts/sharing/SharingClient.kt

```diff
@@ -412,6 +435,23 @@ data class SharingRequest internal constructor(
         ) : Data {
             override fun toString() = "ClipVideo(title=${episode.title}, uuid=${episode.uuid}, start=${range.start.toSecondsWithSingleMilli()}, end=${range.end.toSecondsWithSingleMilli()})"
         }
+
+        class WebLink internal constructor(
+            override val podcast: PodcastModel? = null,
```

Comment:
This can be a computed property instead of a part of the constructor.

Reply:
0a9225c